src/compiler/program.ts

```diff
@@ -1362,18 +1362,16 @@ namespace ts {
             // try to verify results of module resolution
             for (const { oldFile: oldSourceFile, newFile: newSourceFile } of modifiedSourceFiles) {
                 const newSourceFilePath = getNormalizedAbsolutePath(newSourceFile.originalFileName, currentDirectory);
-                if (resolveModuleNamesWorker) {

```

Comment:
Looks like `resolveModuleNamesWorker` is always assigned above, so this was a dead check